src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig

```diff
@@ -1,107 +1,77 @@
+      {% include 'PrestaShopBundle:Admin/Module/Includes:modal_addons_connect.html.twig' with { 'level' : level, 'errorMessage' : errorMessage } %}
+      {# Contains toolbar-nav for module page with level authorization #}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:modal_import.html.twig' with { 'level' : level, 'errorMessage' : errorMessage } %}
+      {# Contains menu with Selection/Categories/Popular and Tag Search #}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:menu_top.html.twig' with { 'topMenuData': topMenuData, 'bulkActions': bulkActions } %}
 
-            <hr class="top-menu-separator">
-
-            <div class="module-short-list">
-                <span id="built-in_modules" class="module-search-result-wording">{{ '%nbModules% built-in modules'|trans({'%nbModules%' : modules.native_modules|length}, 'Admin.Modules.Feature') }}</span>
-                <span class="help-box" data-toggle="popover"
-                  data-title="{{ "Built-in modules"|trans({}, 'Admin.Modules.Feature') }}"
-                  data-content="{{ "These modules from PrestaShop are preinstalled when you set-up your shop. They cover the basics of e-commerce and comes for free."|trans({}, 'Admin.Modules.Help') }}">
-                </span>
-            </div>
-            {% include 'PrestaShopBundle:Admin/Module/Includes:grid_manage_installed.html.twig' with { 'modules': modules.native_modules, 'display_type': 'list', 'origin': 'manage', 'id': 'native' } %}
+      <div id="module-recently-used-list" class="module-short-list">
+        <span id="recently-used_modules" class="module-search-result-wording">{{ 'Recently Used' | trans({}, 'Admin.Modules.Feature') }}</span>
+        <div class="modules-list" data-name="recently-used"></div>
+      </div>
 
-            <hr class="top-menu-separator">
+      {% for category in categories.subMenu %}
+        <div class="module-short-list">
+          <span id="{{ category.tab }}_modules" class="module-search-result-wording">{{ category.name | trans({}, 'Admin.Modules.Feature') }}</span>
 
-            <div class="module-short-list">
-                <span id="theme_modules" class="module-search-result-wording">{{ '%nbModules% theme modules'|trans({'%nbModules%' : modules.theme_bundle|length}, 'Admin.Modules.Feature') }}</span>
-                <span class="help-box" data-toggle="popover"
-                  data-title="{{ "Theme modules"|trans({}, 'Admin.Modules.Feature') }}"
-                  data-content="{{ "Each theme you install will come with its own set of modules. You’ll find here all the modules related to your active theme."|trans({}, 'Admin.Modules.Help') }}">
-                </span>
+          {% if category.modules is empty %}
+            <div class="modules-list module-list-empty" data-name="{{ category.tab }}">
+              <p>
+                {{ 'You do not have module in « %categoryName% ».' | trans({'%categoryName%': category.name}, 'Admin.Modules.Feature') }}<br/>
```

Comment:
What is the behavior we expect here?

Reply:
We want to inform merchants that they don't have any module in the category so they should go to the page "module catalog" to see more products they can add to their store

Reply:
Well, in this case, I would write like that: _You do not have any module in the %category_name% category._